web/Controllers/HomeController.cs

```diff
@@ -30,6 +30,11 @@ public IActionResult Privacy()
     {
         return View();
     }
+    

```

Comment:
Could you add an `[HttpGet]` to this? (and to the other methods in here while you're at it)